# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/linux/hotfix-it87_wdt-init.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- linux-6.17/drivers/watchdog/it87_wdt.c.vanilla	2025-11-06 16:36:17.474866809 +0100
+++ linux-6.17/drivers/watchdog/it87_wdt.c	2025-11-06 16:43:56.756190409 +0100
@@ -374,6 +374,9 @@
 		}
 	}
 
+	/* disable, might be left active by the fw, e.g. ugreen dxp4800 */
+	_wdt_update_timeout(0);
+
 	superio_exit();
 
 	if (timeout < 1 || timeout > max_units * 60) {
